Add note related to 'never,task' rule in auditd (#7291)
wazuh/wazuh-documentation: source/user-manual/capabilities/file-integrity/advanced-settings.rst

         # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
         # rc-service auditd restart

> .. note::

Reviewer: Why a note? Why not just inserting all this in the enabling who-data monitoring steps that follow below?

Author: Paragraph modified with a new layout outside the note!

>    On certain systems, auditd is configured with a default rule that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.

Reviewer: Certain systems such as which ones?

Author: Done

>    Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#top_of_page

Reviewer 2: ```suggestion
   Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
```

Author: done

>    Check in the auditd rules:

Reviewer 2: ```suggestion
   Check auditd rules:
```

Author: done

  
   .. code-block::

      auditctl -l

>       -a never,task

Reviewer: What is `-a never,task` here? Is this the output of the `auditctl -l` command? Is this the real output or an output when piping the grep command? We could use a separate output code-block to show the output. Also, commands in code-blocks need to include the prompt symbol. `code-block` directives need to specify a language for syntax highlighting, for example for Linux command lines we use `.. code-block:: console`.

Author: Done

>    You need to remove that rule, or include a specific filter for the syscheck daemon in the auditd rule file ``/etc/audit/rules.d/audit.rules``:

Reviewer: What's our recommendation? Removing or filtering?

Author: Filtering, done

  
   .. code-block::

      -a always,task -F exe=‘/var/ossec/bin/wazuh-syscheckd’

>       -a never,task

Reviewer: if we are keeping this rule we need to emphasize the filter line with for example `:emphasize-lines: 1` in this code block

Author: Done

         # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
         # rc-service auditd restart

> Check possible DISABLED BY DEFAULT status of auditd
> 
> #. On most systems auditd is configured with a default rule (``-a never,task``) that causes all audit rule processing to be skipped. This disables any whodata-related configuration of Wazuh.
>  
>    Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
>  
>    Check auditd rules:
>  

Reviewer: ```suggestion
In most systems, auditd includes a rule to skip processing of every audit rule by default. This setting prevents the reporting of any whodata-related information. To ensure that auditd is not `DISABLED BY DEFAULT <https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT>`__, follow these steps.

#. Check the output of this command to find out if the auditd rules include the ``-a never,task`` rule.

```

Author: done

   .. code-block:: console
 

>      auditctl -l

Reviewer: ```suggestion
      # auditctl -l | grep task
```

Author: done

> #. If the output shows the above mentioned rule:
>  
>    .. code-block:: console
>  
>      -a never,task
>  
>    You need to include a specific filter for the syscheck daemon, over the other rule, in the auditd rule file ``/etc/audit/rules.d/audit.rules``:

Reviewer: ```suggestion
#. If the output displays the ``-a never,task`` rule, add the following filter rule in ``/etc/audit/rules.d/audit.rules``. Make sure to place it before the mentioned rule.
```

Author: done

>    .. code-block::
>      :emphasize-lines: 1
>  
>      -a always,task -F exe=‘/var/ossec/bin/wazuh-syscheckd’
>      -a never,task

Reviewer: ```suggestion
   .. code-block:: none
      :emphasize-lines: 1
 
      -a always,task -F exe=‘/var/ossec/bin/wazuh-syscheckd’
      -a never,task
```

Author: done

 
#. After that, restart auditd and Wazuh agent to apply the changes:
 
   .. code-block:: console
 
     systemctl restart auditd

>      systemctl restart wazuh-agent

Reviewer: Use three-spaces tabulation

Author: done

> Perform the following steps to enable who-data monitoring. In this example, you configure who-data monitoring for ``/etc`` directory.

Reviewer: ```suggestion
Next, perform the following steps to enable who-data monitoring. In this example, you configure who-data monitoring for the ``/etc/`` directory.
```

Author: done

         # cp /usr/sbin/audisp-af_unix /sbin/audisp-af_unix
         # rc-service auditd restart

In most systems, auditd includes a rule to skip processing of every audit rule by default. This setting prevents the reporting of any whodata-related information. To ensure that auditd is not `DISABLED BY DEFAULT <https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT>`__, follow these steps.

#. Check the output of this command to find out if the auditd rules include the ``-a never,task`` rule.

>    Link to auditd Linux manual: https://man7.org/linux/man-pages/man8/auditctl.8.html#DISABLED_BY_DEFAULT
>  
>    Check auditd rules:
>  

Reviewer: ```suggestion
```
